ui/Dockerfile

```diff
@@ -1,20 +1,14 @@
-FROM node
+FROM nginx:alpine
 
 MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
 
 ARG BUILD_DATE
 ARG VCS_REF
 
-WORKDIR /home/Bassa
+RUN apt-get update 
```

Comment:
Alpine Linux does not use `apt-get` it seems, we have to replace this with an equivalent.